spark/src/test/scala/geotrellis/spark/io/LayerUpdateSpaceTimeTileTests.scala

```diff
@@ -7,17 +7,19 @@ import geotrellis.vector._
 import geotrellis.proj4.LatLng
 
 import org.apache.spark.rdd._
+import org.joda.time.DateTime
 
-trait LayerUpdateSpaceTimeTileTests { self: PersistenceSpec[SpaceTimeKey, Tile, RasterMetaData] with TestEnvironment =>
+trait LayerUpdateSpaceTimeTileTests { self: PersistenceSpec[SpaceTimeKey, Tile, RasterMetaData[SpaceTimeKey]] with TestEnvironment =>
 
   def updater: TestUpdater
 
-  def dummyRasterMetaData: RasterMetaData =
+  def dummyRasterMetaData =
     RasterMetaData(
       IntConstantNoDataCellType,
       LayoutDefinition(RasterExtent(Extent(0,0,1,1), 1, 1), 1),
       Extent(0,0,1,1),
-      LatLng
+      LatLng,
+      KeyBounds(SpaceTimeKey(0,0, new DateTime()), SpaceTimeKey(1,1, new DateTime()))
```

Comment:
Seem like keyBounds are not empty, that causes problems with test `should not update a layer (keys out of bounds)`